apps/ui/src/helpers/connectors/coinbase.ts

```diff
@@ -3,11 +3,9 @@ import Connector from './connector';
 export default class Coinbase extends Connector {
   async connect() {
     try {
-      let CoinbaseWalletSDK = await import('@coinbase/wallet-sdk'!);
-      if (CoinbaseWalletSDK.default)
-        CoinbaseWalletSDK = CoinbaseWalletSDK.default;
-      if (CoinbaseWalletSDK.default)
-        CoinbaseWalletSDK = CoinbaseWalletSDK.default;
+      const { default: CoinbaseWalletSDK } = await import(
+        '@coinbase/wallet-sdk'
+      );
```

Comment:
Maybe we can import CoinbaseWalletSDK directly instead of `default` https://docs.cdp.coinbase.com/wallet-sdk/docs/initializing#initializing